spec/requests/api/v1/users/login_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+require "rspec/json_expectations"
+
+describe 'Login User', type: :request do
+  subject { post '/api/v1/auth/sign_in', params:, as: :json }
+
+  before(:each) do
+    @current_user = FactoryBot.create(:user)
+  end
```

Comment:
```suggestion
  let!(:user) { create(:user) } 
```
then you should replace the `@current_user` for `user` in each place